tests/assets/third-party-compatibility/requirejs/2.3.6.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2022 New Relic Corporation.
+  PDX-License-Identifier: Apache-2.0
+-->
+<html>
+  <head>
+    <title>RequireJS 2.3.6</title>
+    <!--  RequireJS is a JavaScript file and module loader.
+          The data-main attribute tells require.js to load JQuery after
+          RequireJS loads. For other typical usages, see the docs:
+          https://requirejs.org/docs/start.html  -->
+    <!-- <script data-main="js/vendor/jquery-3.2.1" src="js/vendor/require-2.3.6.js"></script> -->
+    <script data-main="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.0/jquery.js" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.js" integrity="sha512-vRqhAr2wsn+/cSsyz80psBbCcqzz2GTuhGk3bq3dAyytz4J/8XwFqMjiAGFBj+WM95lHBJ9cDf87T3P8yMrY7A==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```

Comment:
in the same vein as mootools, do we need to draw a line in the sand for requireJS versions?

Reply:
Not sure. This is the latest version of requirejs. I don't know how far back we want to, care to, can test.